src/modules/app/StoreActionType.js

```diff
@@ -1,180 +1,224 @@
-  |}
-|}
-
-export type ClearCategoryActionType = {|
-  type: 'CLEAR_CATEGORY', +params: {| +key: string |}
-|}
-
-export type CategoriesActionType =
-  ClearCategoryActionType
-  | FetchCategoryActionType
-  | PushCategoryActionType
-  | FetchCategoryFailedActionType
-
-export type FetchEventActionType = {|
-  type: 'FETCH_EVENT', +params: {|
-    +city: string, +language: string,
-    +path: ?string, +key: string,
-    +criterion: ContentLoadCriterionType
-  |}
-|}
-export type ClearEventActionType = {|
-  type: 'CLEAR_EVENT', +params: {| +key: string |}
-|}
-export type PushEventActionType = {|
-  type: 'PUSH_EVENT',
-  +params: {|
-    +events: $ReadOnlyArray<EventModel>,
-    +path: ?string,
-    +key: string,
-    +resourceCache: LanguageResourceCacheStateType,
-    +cityLanguages: $ReadOnlyArray<LanguageModel>,
-    +language: string,
-    +city: string
-  |}
-|}
-export type FetchEventFailedActionType = {|
-  type: 'FETCH_EVENT_FAILED',
-  +params: {|
-    +message: string,
-    +code: ErrorCodeType,
-    +key: string,
-    +allAvailableLanguages: ?$ReadOnlyMap<string, ?string>,
-    +language: string,
-    +path: ?string,
-    +city: string
-  |}
-|}
-
-export type EventsActionType =
-  ClearEventActionType
-  | FetchEventActionType
-  | PushEventActionType
-  | FetchEventFailedActionType
-
-export type SwitchContentLanguageActionType = {|
-  type: 'SWITCH_CONTENT_LANGUAGE', +params: {|
-    // The TFunction should be removed again in https://issues.integreat-app.de/browse/NATIVE-359
-    +newLanguage: string, +city: string, +t: TFunction
-  |}
-|}
-
-export type SwitchContentLanguageFailedActionType = {|
-  type: 'SWITCH_CONTENT_LANGUAGE_FAILED', +params: {|
-    +message: string
-  |}
-|}
-
-export type ContentLanguageActionType = SwitchContentLanguageActionType | SwitchContentLanguageFailedActionType
-
-export type MorphContentLanguageActionType = {|
-  type: 'MORPH_CONTENT_LANGUAGE', +params: {|
-    +newCategoriesMap: CategoriesMapModel,
-    +newResourceCache: LanguageResourceCacheStateType,
-    +newEvents: $ReadOnlyArray<EventModel>,
-    +newLanguage: string
-  |}
-|}
-
-export type ClearCityActionType = {|
-  type: 'CLEAR_CITY'
-|}
-
-export type ResourcesFetchFailedActionType = {|
-  type: 'FETCH_RESOURCES_FAILED',
-  +params: {|
-    +message: string,
-    +code: ErrorCodeType
-  |}
-|}
-
-export type CityContentActionType =
-  CategoriesActionType
-  | EventsActionType
-  | MorphContentLanguageActionType
-  | ContentLanguageActionType
-  | ClearCityActionType
-  | PushLanguagesActionType
-  | FetchLanguagesFailedActionType
-  | ResourcesFetchFailedActionType
-
-export type ToggleDarkModeActionType = {|
-  type: 'TOGGLE_DARK_MODE'
-|}
-
-export type ClearResourcesAndCacheActionType = {|
-  type: 'CLEAR_RESOURCES_AND_CACHE'
-|}
-
-export type StoreActionType =
-  ToggleDarkModeActionType
-  | CitiesActionType
-  | CityContentActionType
-  | SetContentLanguageActionType
-  | ClearResourcesAndCacheActionType
+// @flow

```

Comment:
Why did you change the mode of this file from 644 to 755? Please revert this. Else the diff it not easy to check.